Gemfile

```diff
@@ -8,6 +8,7 @@ group :development, :unit_tests do
   gem 'simplecov', :require => false
   gem 'puppet_facts', :require => false
   gem 'json', :require => false
+  gem 'metadata-json-lint', :require => false
```

Comment:
I know, I just wanted to pass Travis tests (won't be necessary if you would run linting and validations after specs).